terraform-provider-constellation/internal/provider/cluster_resource.go

```diff
@@ -1240,15 +1240,15 @@ type tfContextLogger struct {
 	ctx context.Context // bind context to struct to satisfy interface
 }
 
-func (l *tfContextLogger) Debugf(format string, args ...any) {
+func (l *tfContextLogger) Debug(format string, args ...any) {
 	tflog.Debug(l.ctx, fmt.Sprintf(format, args...))
```

Comment:
I don't fully see through this. Are args here still values to be inserted into a format string, or are they key-value pairs of slog?

Reply:
The arguments here are still inserted into a format string as it was done before since the `tfContextLogger` doesn't use `slog`. This code is weird because `tfContextLogger` has to implement `debugLog`, which also has to be implemented by the `slog.Logger` in other parts of the code.

Reply:
Got it, thanks. Please add a short comment to avoid confusion.